tips/TIP-0048/openapi3-indexer.yaml

```diff
@@ -0,0 +1,1021 @@
+              schema:
+                $ref: '#/components/schemas/BadRequestResponse'
+        '403':
+          description: "Unsuccessful operation: indicates that the endpoint is not available for public use."
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ForbiddenResponse'
+        '500':
+          description: "Unsuccessful operation: indicates that an unexpected, internal server error happened which prevented the node from fulfilling the request."
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/InternalErrorResponse'
+ 
+  /api/indexer/v2/outputs/delegation/{delegationId}:
+    get:
+      tags:
+        - delegation outputs
+      summary: Returns the delegations for the delegationId.
+      description: Returns the delegations for the delegationId.
+      parameters:
+        - in: path
+          name: delegationId
+          schema:
+            type: string
+          example: "0x1505ec099896ab05d9e08fbc7101ae4dff0093b3943b28f789ed2ca728bcc8d6"
+          description: Unique identifier of the delegationId.
+          required: true
+      responses:
+        '200':
+          description: "Successful operation."
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/OutputsResponse'
+              examples:
+                Query results in single output:
+                  $ref: >-
+                    #/components/examples/get-outputs-response-single-example
+        '400':
+          description: "Unsuccessful operation: indicates that the provided data is invalid."
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/BadRequestResponse'
+        '403':
+          description: "Unsuccessful operation: indicates that the endpoint is not available for public use."
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ForbiddenResponse'
+        '404':
+          description: "Unsuccessful operation: indicates that the requested data was not found."
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/NotFoundResponse'
+        '500':
+          description: "Unsuccessful operation: indicates that an unexpected, internal server error happened which prevented the node from fulfilling the request."
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/InternalErrorResponse'
+
+  /api/indexer/v2/outputs/foundry:
+    get:
+      tags:
+        - foundry outputs
+      summary: Returns foundry outputs filtered based on parameters.
+      description: Returns foundry outputs filtered based on parameters.
+      parameters:
+        - in: query
+          name: accountAddress
```

Comment:
This seems to be just `account` in the go code, what is correct ? https://github.com/iotaledger/inx-indexer/blob/67aa80d647a34f4184232d862bf0fec12ba18965/pkg/indexer/foundry.go#L31
I have a preference for `accountAddress`

Reply:
yeah, it's `account`.